bin/translateAttributes.php

```diff
@@ -32,7 +32,7 @@
 
 $attributes = [];
 
-$languages = SimpleSAML\Locale\Language::$language_names;
+$languages = Symfony\Component\Intl\Languages::getNames();
```

Comment:
without a `use` above wouldn't this need a leading `\`, ie `\Symfony\Component....`

Reply:
Nope, not unless the file starts with a `namespace` statement, which it doesn't.

Reply:
ah ok. cool :)